Sources/VLCAlertControllerExtension.swift

```diff
@@ -0,0 +1,109 @@
+/*****************************************************************************
+ * VLCAlertControllerExtension.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Pierre Sagaspe <pierre.sagaspe # me.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+typealias UIAlertCompletionBlock = (UIAlertController, Int) -> Void
+typealias UIAlertTextFieldCompletionBlock = (UIAlertController, Int, String) -> Void
+
+struct buttonKeys {
```

Comment:
This struct seems to be only used here internally, what do you think about defining it as `private`?

Anyway, since it defines a date type, I think it should be uppercased.